pkgs/by-name/tf/tftui/package.nix

```diff
@@ -19,15 +19,16 @@ python3.pkgs.buildPythonApplication rec {
   };
 
   pythonRelaxDeps = [
+    "posthog"
     "textual"
   ];
 
-  nativeBuildInputs = with python3.pkgs; [
+  nativeBuildInputs = with python3Packages; [
     makeWrapper
     poetry-core
   ];
 
-  propagatedBuildInputs = with python3.pkgs; [
+  propagatedBuildInputs = with python3Packages; [
```

Comment:
```suggestion
  dependencies = with python3Packages; [
```

Reply:
As long as I was smashing nits, all those `with python3Packages` had to go.